draft-ietf-trans-rfc6962-bis.md

```diff
@@ -626,26 +626,48 @@ not incorporated in the issued certificate is when a CA sends the precertificate
 to multiple logs, but only incorporates the SCTs that are returned first.
 
 A precertificate is a CMS [RFC5652] `signed-data` object that conforms to the
-following requirements:
+following profile:
 
 * It MUST be DER encoded.
 
-* `SignedData.encapContentInfo.eContentType` MUST be the OID 1.3.101.78.
-
-* `SignedData.encapContentInfo.eContent` MUST contain a TBSCertificate [RFC5280]
-  that will be identical to the TBSCertificate in the issued certificate, except
-  that the Transparency Information ({{x509v3_transinfo_extension}}) extension
-  MUST be omitted.
-
-* `SignedData.signerInfos` MUST contain a signature from the same (root or
-  intermediate) CA that will ultimately issue the certificate. This signature
-  indicates the CA's intent to issue the certificate. This intent is considered
-  binding (i.e., misissuance of the precertificate is considered equivalent to
-  misissuance of the certificate). (Note that, because of the structure of CMS,
-  the signature on the CMS object will not be a valid X.509v3 signature and so
-  cannot be used to construct a certificate from the precertificate).
-
-* `SignedData.certificates` SHOULD be omitted.
+* `SignedData.version` MUST be v3(3).
+
+* `SignedData.digestAlgorithms` MUST only include the
+  `SignerInfo.digestAlgorithm` OID value (see below).
+
+* `SignedData.encapContentInfo`:
+  * `eContentType` MUST be the OID 1.3.101.78.
+  * `eContent` MUST contain a TBSCertificate [RFC5280] that will be identical to
+    the TBSCertificate in the issued certificate, except that the Transparency
+    Information ({{x509v3_transinfo_extension}}) extension MUST be omitted.
+
+* `SignedData.certificates` MUST be omitted.
+
+* `SignedData.crls` MUST be omitted.
+
+* `SignedData.signerInfos` MUST contain one `SignerInfo`:
+  * `version` MUST be v3(3).
+  * `sid` MUST use the `subjectKeyIdentifier` option.
+  * `digestAlgorithm` MUST be one of the hash algorithm OIDs listed in
+    {{hash_algorithms}}.
+  * `signedAttrs` MUST be present and MUST contain two attributes:
+    * A content-type attribute whose value is the same as
+      `SignedData.encapContentInfo.eContentType`.
+    * A message-digest attribute whose value is the message digest of
```

Comment:
I think RFC5652 section 5.3 is clear that the message-digest attribute has to be explicitly present in signedAttrs:
`signedAttrs...If the field is present, it MUST contain, at a minimum...A message-digest attribute.`